### ISSUE TITLE
Observe Calorie History: UI Edition

### ISSUE THREAD

**Issue by Brahmancer** [ScienceSquad/Health#9]
Implement a calorie intake history such that the user can see it in the form of a log and a graph.  

---
- **Estimated Time:** _9 hours._
- **Prerequisites:** _Calendar View, Nutrition Module._ _Nutrition ViewModel_

**Comment by MrJohnson** [comment]
I wonder if we should move this issue to Sprint 2. I think Sprint 1 should be focused on core functionality, and Sprint 2 will focus more on data analysis. However, this particular feature does seem fairly straightforward once nutrition logging is implemented.

**Comment by avaidyam** [comment]
This also aligns pretty well with the "calendar" or "summary" view we had in mind with issue #40. Sprint #3 might be better. Any thoughts @MrJohnson @Brahmancer?

**Comment by Brahmancer** [comment]
I think we should start looking at this in Sprint 2. What I'm going to do during this sprint and essentially get Realm set up on one of the Modules (Nutrition) so I can see how one module will interact with it's realm. Once I got it working, I'll create a wiki page so that everyone creating a module can set it up in the same way. With that set up, we can start finding ways on how we would store days in the database then start pulling from it to create the calendar. However, this can be put off if other portions of the application seem more important. 

TL;DR: The logging will be set up by the end of this sprint. So we can start looking at creating some calendar/ history of Module data in the next sprint. If we find that other things take precedent, we can push this off, imho.

**Comment by avaidyam** [comment]
:+1: Just moved it over. I was taking a good look at apps like Jawbone's UP, and they have REALLY nice summary and "tends" views. I'll go over those in our next meeting.

**Comment by Brahmancer** [comment]
This would be more of graphical data history then?

**Comment by Brahmancer** [comment]
#87
